src/background/sidebar-injector.js

```diff
@@ -323,14 +323,7 @@ export default function SidebarInjector(
    */
   function injectConfig(tabId, config) {
     const configStr = JSON.stringify(config).replace(/"/g, '\\"');
-    const configCode =
-      'var hypothesisConfig = "' +
-      configStr +
-      '";\n' +
-      '(' +
-      addJSONScriptTagFn.toString() +
-      ')' +
-      '("js-hypothesis-config", hypothesisConfig);\n';
+    const configCode = `var hypothesisConfig="${configStr}";\n(${addJSONScriptTagFn})("js-hypothesis-config", hypothesisConfig);\n`;
```

Comment:
I can't remember why a variable was used here originally. We're going to have to change this soon anyway for Manifest V3 support, because Manifest V3 does not allow the execution of code strings.